Include/internal/pycore_frame.h

```diff
@@ -101,7 +103,7 @@ _PyFrame_InitializeSpecials(
     frame->generator = NULL;
     frame->f_lasti = -1;
     frame->f_state = FRAME_CREATED;
-    frame->depth = 0;
+    frame->own_cframe = true;
```

Comment:
This should default to `false`. Then it only need be set at the start of `_PyEval_EvalFrameDefault`, not for every internal call.